### PR TITLE
Fixed not translating auto-evo prediction texts

### PR DESCRIPTION
**Pull request by takko**
**Brief Description of What This PR Does**

Extracted the auto evo prediction text fixes from this PR #2790.

> This picture made by Kasterix, shows that "Pangonian Vents" and "N/A" are not updated when the language is changed as well: 142730979-9e7ec734-0228-462d-ba9c-4a427db8635c

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: src/microbe_stage/editor/MicrobeEditorGUI.cs [hunk 1]
```diff
@@ -1207,6 +1222,41 @@ public void OnDeletePressed()
...
+        if (what == NotificationTranslationChanged)
+        {
+            if (autoEvoRunSuccessful.HasValue && autoEvoRunSuccessful.Value == false)
+            {
+                totalPopulationLabel.Text = TranslationServer.Translate("FAILED");
+            }
+
+            if (!string.IsNullOrEmpty(bestPatchName))
+            {
+                bestPatchLabel.Text = string.Format(CultureInfo.CurrentCulture,
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/MicrobeEditorGUI.cs:1236]: I'd like to see a common method be created where this duplicate code would be placed.

File: src/microbe_stage/editor/MicrobeEditor.cs [hunk 2]
```diff
@@ -654,8 +654,6 @@ public override void _Notification(int what)
             gui.UpdatePatchDetails(CurrentPatch);
             gui.UpdateMicrobePartSelections();
             gui.UpdateMutationPointsBar();
-
-            // TODO: AutoEvo run results summary
```
Comment (Kasterisk) [inline src/microbe_stage/editor/MicrobeEditor.cs:658]: Shouldn't this be the one fixed in #2790?

**Review by Kasterisk — CHANGES_REQUESTED** [review]

File: src/microbe_stage/editor/MicrobeEditor.cs (hunk 2 above)
Comment (takko) [reply src/microbe_stage/editor/MicrobeEditor.cs]: true, moved it.

**Comment by hhyyrylainen** [comment]
Could someone test this? If this works this can be merged in for 0.5.6.1 already.

**Comment by RacerBG** [comment]
> Could someone test this? If this works this can be merged in for 0.5.6.1 already.

Give me amoment, I just need to figure out the switch to this PR. :smiley:

**Comment by RacerBG** [comment]
It works. Sorry for taking so long, I had to RTFM before using git.

**Review by hhyyrylainen — APPROVED** [review]
Code looks fine and loading an older save didn't seem to explode anything.

**Review by Kasterisk — APPROVED** [review]

**Pull request merged**